demos/object_detection_demo/jupyter-python/object_detection_demo.ipynb

```diff
@@ -0,0 +1,750 @@
+    "PROB_THRESHOLD = 0.5  # The probability threshold for detection predictions\n",
+    "DEFAULT_NUM_THREADS = 3  # Default number of threads to use for inference\n",
+    "DEFAULT_NUM_STREAMS = 3  # Default number of streams to use for inference\n",
+    "DEFAULT_NUM_REQUESTS = 4  # Default maximum number of requests to use for inference\n",
+    "\n",
+    "## Visualization settings\n",
+    "PALETTE = ColorPalette(100)\n",
+    "FONT_SCALE = 1\n",
+    "THICKNESS = 2\n",
+    "\n",
+    "# The settings below are only required if you want to use the Model Converter to convert models to OpenVINO IR format.\n",
+    "# You can use this demo with models that are already downloaded in IR format, so use of the model optimizer is optional.\n",
+    "\n",
+    "# The path to the Model Optimizer is required if models need to be converted to IR. The paths below should work for default installations of\n",
+    "# the Intel Distribution of OpenVINO Toolkit https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html\n",
+    "# Adjust them if you installed OpenVINO in a different location.\n",
+    "# Note that you also need to install the Model Optimizer prerequisites. See the documentation for your OS at\n",
+    "# https://docs.openvinotoolkit.org/latest/installation_guides.html\n",
+    "\n",
+    "CONVERT_MODELS = False  # Set to True to use public Open Model Zoo models and convert them with the Model Optimizer.\n",
+    "if CONVERT_MODELS:\n",
+    "    if sys.platform.startswith(\"win\"):  # Windows\n",
+    "        model_optimizer_path = r\"C:\\Program Files (x86)\\intel\\openvino_2021\\deployment_tools\\model_optimizer\\mo.py\"\n",
+    "    else:  # Linux/MacOS\n",
+    "        model_optimizer_path = \"/opt/intel/openvino_2021/deployment_tools/model_optimizer/mo.py\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "powerful-valuable",
+   "metadata": {},
+   "source": [
+    "## Download Models and convert them to IR format\n",
+    "\n",
+    "The [Model Downloader](https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/downloader/README.md) downloads models from the Open Model Zoo. Models that are not in OpenVINO IR format are converted to this format by the Model Converter. \n",
+    "\n",
+    "A subset of [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/) models that are compatible with this demo are listed in the file `models_file` (default \"models.lst\") in the same folder as this notebook. By default these models are downloaded, with the `--list=models_file` argument for the Model Downloader. You can choose to download a specific model by using `--name=model_name` instead of `--list=models.lst`. If you already have downloaded Open Zoo Models, you can set the `base_model_dir` variable in the *Settings* cell to the folder that contains your models (this should be a folder with subfolders `intel` and `public`) and set `DOWNLOAD_MODELS` to `False`.\n",
+    "\n",
```

Comment:
```suggestion
    "\n",
    "Note that [Model Downloader](https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/downloader/README.md) and Model Converter are standalone tools and can be executed using command line. Models have to be downloaded only once. Thus the code below (calling those tools) is usually not included into real applications as they assume that model is already downloaded and converted (or provided in application package)."
    "\n",
```

Reply:
I like the first part, that is a good clarification. Would you be okay with leaving out this part "Thus the code below (calling those tools) is usually not included into real applications as they assume that model is already downloaded and converted (or provided in application package"? A notebook is always a bit different than a "real application". I also show a message that "DOWNLOAD_MODELS" can be set to False once it has been run.

Reply:
We can rephrase second part. I just wanted to emphasize that in most application that part of python code would not be used because model is either already converted or downloader/converter are used as standalone tools. That would give customers a hint that this part of code should not be copied directly to their apps, as other parts. 
But if we cannot rephrase it - I'm OK with leaving it out.